pkg/plugins/golang/deploy-image/v1alpha1/scaffolds/api.go

```diff
@@ -109,6 +109,14 @@ func (s *apiScaffolder) Scaffold() error {
 		return fmt.Errorf("error updating main.go: %v", err)
 	}
 
+	if err := s.updateManagerWithNewEnvKey(); err != nil {
+		return fmt.Errorf("error updating config/manager/manager.yaml with env key: %v", err)
+	}
+
+	if err := s.updateManagerWithNewEnvVar(); err != nil {
+		return fmt.Errorf("error updating config/manager/manager.yaml with env var: %v", err)
+	}
```

Comment:
it seems duplicated

Reply:
No, @camilamacedo86 these are different. The former one is to scaffold the `env` key only and the second one is to scaffold the `name` and `value` of the `env` var. I did them in two different functions because we only want `env:` to be scaffolded one but the `name` and `value` can be scaffolded multiple times if multiple APIs are scaffolded.

Could you please give your thoughts.

Reply:
Ok, I am taking a look at it to make a better approach to dealing with it.

Reply:
Thank you for the explanation. I see now, it makes sense. Good work